### PR TITLE
Rename RTCIceCandidateStats.ip to address.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fix for https://github.com/w3c/webrtc-pc/issues/1913.

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -2953,13 +2953,13 @@ <h2>
...
               <dt>
-                <dfn><code>ip</code></dfn> of type <span class=
+                <dfn><code>address</code></dfn> of type <span class=
                 "idlMemberType"><a>DOMString</a></span>
               </dt>
               <dd>
                 <p>
-                  It is the IP address of the candidate, allowing for IPv4 addresses and IPv6
-                  addresses, but fully qualified domain names (FQDNs) are not allowed. See
+                  It is the IP address of the candidate, allowing for IPv4 addresses, IPv6
```
Comment (fippo) [inline webrtc-stats.html]: is the address of the candidate

Comment (fippo) [reply webrtc-stats.html]: oh that is the prose from 5245... which no longer seems to be in the successor? anyway, :+1: on using that prose for consistency

Comment (jan-ivar) [reply webrtc-stats.html]: As discussed offline, this matches the language in [rfc5245](https://tools.ietf.org/html/rfc5245#section-15.1). I suppose it has a bit of a terminology issue. Anyone feel strongly which way it should be here?

Comment (jan-ivar) [reply webrtc-stats.html]: Turns out others agree with you, so I made the change. Thanks!

**Comment by vr000m** [comment]
LGTM

**Comment by jan-ivar** [comment]
@vr000m Thanks, can we merge this now that https://github.com/w3c/webrtc-pc/pull/1985 has merged?

**Comment by vr000m** [comment]
Done. I was waiting for the webrtc-pc PR to be merged. With that done.

**Pull request merged**